games/snake/snake_input_console.h

```diff
@@ -63,6 +63,27 @@ void init_termios(int echo)
   fcntl(STDIN_FILENO, F_SETFL, O_NONBLOCK);  /* set non-blocking mode */
 }
 
+/****************************************************************************
+ * Name: deinit_termios
+ *
+ * Description:
+ *   Restore back g_old terminal I/O settings.
+ *
+ * Parameters:
+ *   None
+ *
+ * Returned Value:
+ *   None.
+ *
+ ****************************************************************************/
+
+void deinit_termios(void)
+{
+  g_old.c_lflag |= ECHO;              /* enable ECHO bit */
```

Comment:
the previous setting may disable echo too, so it's better to restore g_old directly